controllers/index.js

```diff
@@ -20,11 +31,14 @@ module.exports = {
         } else {
           query.$text = { $search: req.query.keywords }
         }
+        return query
```

Comment:
This is making the response hang for me!

Reply:
my fault! 🤦